Plugins/CoreAPI/server/api-functions.js

```diff
@@ -0,0 +1,488 @@
+import PenPal from "meteor/penpal";
```

Comment:
As a point of organization I personally would prefer we put this in a folder and separate functions by schema Type they affect. Like:

```
Plugins/CoreAPI/server/api/
 |--> index.js
 |--> hosts.js
 |--> projects.js
 |--> services.js
```

Inside `index.js` you can `export default` the `API` object with fields for `Hosts`, etc based on imports from the individual files.

Reply:
Sounds good, will make that change as we do some initial plugins...